src/styleguide/routes/map.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <l-default>
+    <c-google-map :locations="locations" theme="gray" :class="b('google-map')" />
```

Comment:
todo: please always define static attributes (`theme="gray"`) AFTER dynamic ones `:locations="locations"`).

Reply:
@patric-eberle Don't you mean BEFORE ? By defining static attributes first, the virtual DOM can more easily identify changes in the dynamic attributes. This can help to improve the performance of your application and make it more efficient.

It's also worth noting that when you are using class binding with a ternary operator it's also a good practice to define the static class first and then the dynamic one. This way you can ensure that the static class will be always present and the dynamic one will be added or removed depending on the condition.